_posts/2020-02-16-walking.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: Walking Is A Choice
+date: 2020-02-15 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [General]
+image:
+image2:
+author: Valentino Urbano
+---
+
+## Walking
+
+Walking is a choice.
+
+I have an office job and if I wanted to I could just sit all day in my chair without moving at all (assuming I do not have meetings that day). For some time I spent the time at my job like that, like I expect most people do as well.
```

Comment:
For a while I spent (...)
first job? 
second like > as

Reply:
Just any other job without an office like this I would... but yeah